vtex/faststore
# Migrate ShippingSimulator component to @vtex/store-ui

## What's the purpose of this pull request?



In order for us to provide a more stable API, visual components that are currently at `gatsby-theme-store` should be moved to the `@vtex/store-ui` package. The idea is that `gatsby-theme-store` should be the skeleton of a store, while `@vtex/store-ui` is more of a component library.

This PR does that for the `ShippingSimulator` component.

## How it works? 


This is not a new feature, but rather a migration. Changes that could be noticed in the API are the following:

- The path `gatsby-theme-store/components/ShippingSimulator` now only contains an `index.tsx` file, and `hooks/useShippingSimulator.ts`. Imports for anything other than these files will stop working.
- `@vtex/store-ui` now exports a `ShippingSimulator` component.

## How to test it?


1. Checkout to this branch locally.
2. Run `yarn link` at `packages/store-ui` and `packages/gatsby-theme-store`.
3. Go to any store's theme, such as `marinbrasil.store`.
4. Run `yarn link @vtex/store-ui` and `@vtex/gatsby-theme-store`.
5. Now just run `yarn develop` and you should be able to access Marin Brasil at `localhost:8000`.
6. Open any product's page and interact with the `ShippingSimulator` component.
